cudamapper/src/overlapper_triggered.cu

```diff
@@ -137,18 +137,31 @@ struct FilterOverlapOp
 {
     size_t min_residues;
     size_t min_overlap_len;
+    size_t min_bases_per_residue;
+    float min_overlap_fraction;
 
-    __host__ __device__ __forceinline__ FilterOverlapOp(size_t min_residues, size_t min_overlap_len)
+    __host__ __device__ __forceinline__ FilterOverlapOp(size_t min_residues, size_t min_overlap_len,
```

Comment:
I'm not sure. @akkamesh ?

Reply:
I'm rather sure it is not needed

Reply:
It appears to be needed:

```
/home/mike/dev/ClaraGenomicsAnalysis/cudamapper/src/overlapper_triggered.cu(397): error: no suitable constructor exists to convert from "size_t" to "claragenomics::cudamapper::FilterOverlapOp"
```

Reply:
Even if you use `FilterOverlapOp filterOp{min_residues, min_overlap_len, min_bases_per_residue, min_overlap_fraction};` instead of `FilterOverlapOp filterOp(min_residues, min_overlap_len, min_bases_per_residue, min_overlap_fraction);` on the line where it fails?
Anyway, this is not crucial so we can leave it as it is.